packages/messages/src/browser/notifications.ts

```diff
@@ -72,8 +74,18 @@ export class Notifications {
                 button.addEventListener('click', () => {
                     action.fn(handler);
                     close();
+                    clearTimeout(this.closeTimer);
                 });
             }
+
+            // Do not auto-close the "error" notification
+            if (properties.icon.toLowerCase() !== 'error') {
```

Comment:
it should be configurable via properties of a `Message` as required in  https://github.com/theia-ide/theia/issues/1081#issue-289915333, because the client needs to decide, if this message should be sticky or not.